platform/commonUI/notification/src/NotificationService.js

```diff
@@ -58,21 +58,23 @@ define(
          * @property {string} title The title of the message
          * @property {string} severity The importance of the message (one of
          * 'info', 'alert', or 'error' where info < alert <error)
-         * @property {number} progress The completion status of a task
+         * @property {number} [progress] The completion status of a task
          * represented numerically
-         * @property {boolean} unknownProgress a boolean indicating that the
+         * @property {boolean} [unknownProgress] a boolean indicating that the
          * progress of the underlying task is unknown. This will result in a
          * visually distinct progress bar.
-         * @property {boolean | number} autoDismiss If truthy, dialog will
+         * @property {boolean | number} [autoDismiss] If truthy, dialog will
          * be automatically minimized or dismissed (depending on severity).
          * Additionally, if the provided value is a number, it will be used
          * as the delay period before being dismissed.
-         * @property {NotificationOption} primaryOption the default user
+         * @property {boolean} [dismissable=true] If truthy, notification will
```

Comment:
Very much a nit, but think this should read "if true" rather than "if truthy"; the latter softly implies that non-boolean values will behave in a certain way, but really non-boolean values are just bad input here. (In particular, the check is `dismissable !== false` which treats `undefined` or `null` the same as `true`.)